api/src/api/users/user_routes.py

```diff
@@ -0,0 +1,49 @@
+import logging
+
+from flask import Response
+
+from src.api import response
+from src.api.route_utils import raise_flask_error
+from src.api.users import user_schemas
+from src.api.users.user_blueprint import user_blueprint
+from src.auth.api_key_auth import api_key_auth
+
+logger = logging.getLogger(__name__)
+
+# Descriptions in OpenAPI support markdown https://swagger.io/specification/
+SHARED_ALPHA_DESCRIPTION = """
+__ALPHA VERSION__
+
+This endpoint in its current form is primarily for testing and feedback.
+
+Features in this endpoint are still under heavy development, and subject to change. Not for production use.
+
+See [Release Phases](https://github.com/github/roadmap?tab=readme-ov-file#release-phases) for further details.
+"""
+
+
+@user_blueprint.post("/token")
+@user_blueprint.input(
+    user_schemas.UserTokenHeaderSchema, location="headers", arg_name="x_oauth_login_gov"
+)
+@user_blueprint.output(user_schemas.UserTokenResponseSchema)
+@user_blueprint.auth_required(api_key_auth)
+def user_token(x_oauth_login_gov: dict) -> response.ApiResponse | Response:
```

Comment:
Don't need `Response` - that's on the search endpoint due to CSV file stuff it can do.

```suggestion
def user_token(x_oauth_login_gov: dict) -> response.ApiResponse:
```